django-stubs/db/models/manager.pyi

```diff
@@ -121,10 +122,23 @@ class Manager(BaseManager[_T]): ...
 
 class RelatedManager(Manager[_T]):
     related_val: Tuple[int, ...]
-    def add(self, *objs: Union[_T, int], bulk: bool = ...) -> None: ...
+    def add(
+        self,
+        *objs: Union[_T, int],
+        bulk: bool = ...,
+        through_defaults: Optional[Union[Dict[str, Any], Callable]] = None...,
```

Comment:
Is it just `Callable`? Or can we be more specific?

Reply:
The Django docs generically state that as of 3.1, `through_defaults` can be a callable: https://docs.djangoproject.com/en/3.2/ref/models/relations/#django.db.models.fields.related.RelatedManager.add